### ISSUE TITLE
Exception (thepiratebay): Connection refused: Connection refused

### ISSUE THREAD

**Issue by thestaffie** [Jackett/Jackett#5687]
I keep on getting 
_An error occured while testing this indexer
Exception (thepiratebay): Connection refused: Connection refused_

![Jackett 2019-07-19 11-18-43](https://user-images.githubusercontent.com/52735977/61524587-036dad80-aa17-11e9-98ff-43fad8958b14.png)

How do I fix it?
**Message**
Unable to connect to indexer: Torznab error detected: Jackett.Common.IndexerException: Exception (thepiratebay): Connection refused ---> System.Net.Http.HttpRequestException: Connection refused ---> System.Net.Sockets.SocketException: Connection refused
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at CloudFlareUtilities.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs:line 241
   at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/WebClient.cs:line 114
   at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 445
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1230
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 331
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 351
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 815
   at Jackett.Server.Controllers.ResultsController.Torznab(TorznabRequest request) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/ResultsController.cs:line 367

**Exception**
NzbDrone.Core.Indexers.Torznab.TorznabException: Torznab error detected: Jackett.Common.IndexerException: Exception (thepiratebay): Connection refused ---> System.Net.Http.HttpRequestException: Connection refused ---> System.Net.Sockets.SocketException: Connection refused
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at CloudFlareUtilities.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs:line 241
   at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/WebClient.cs:line 114
   at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 445
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1230
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 331
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 351
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 815
   at Jackett.Server.Controllers.ResultsController.Torznab(TorznabRequest request) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/ResultsController.cs:line 367
  at NzbDrone.Core.Indexers.Torznab.TorznabRssParser.PreProcess (NzbDrone.Core.Indexers.IndexerResponse indexerResponse) [0x000b6] in C:\BuildAgent\work\5d7581516c0ee5b3\src\NzbDrone.Core\Indexers\Torznab\TorznabRssParser.cs:42 
  at NzbDrone.Core.Indexers.RssParser.ParseResponse (NzbDrone.Core.Indexers.IndexerResponse indexerResponse) [0x0000d] in C:\BuildAgent\work\5d7581516c0ee5b3\src\NzbDrone.Core\Indexers\RssParser.cs:55 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].FetchPage (NzbDrone.Core.Indexers.IndexerRequest request, NzbDrone.Core.Indexers.IParseIndexerResponse parser) [0x00008] in C:\BuildAgent\work\5d7581516c0ee5b3\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:302

**Comment by garfield69** [comment]
use your jackett dashboard to edit TPB config and report back the **Site Link**.
also specify the platform you are running jackett on, and the jackett version number.
finally report whether you are using the default jackett configuration, or if you are using jackett options such as proxies, alternate webclients etc.

**Comment by thestaffie** [comment]
> use your jackett dashboard to edit TPB config and report back the **Site Link**.  **_https://thepiratebay.org/_**
> also specify the platform you are running jackett on, and the jackett version number. **_Ubuntu Jackett Version 0.11.507.0_**
> finally report whether you are using the default jackett configuration, or if you are using jackett options such as proxies, alternate webclients etc. **_default jackett configuration_**

**Comment by thestaffie** [comment]
@garfield69 I am a noob at this and trying to help myself as I go along... so patience please

![image](https://user-images.githubusercontent.com/52735977/61631949-00c7be00-ac8c-11e9-8c71-801bc6b46521.png)

Just cant get those two to work. Want to use Sonarr for my Series and Couchpotato for movies but it seems that both are not optimally set and I am lost...

**Comment by garfield69** [comment]
so from your jackett dashboard, click on the wrench icon on the row for thepiratebay and report back the Site Link in the config box. do the same the kickasstorrent.
also scroll down to the bottom of the dashboard and capture the jackett configuration section, I just need to see that your jackett setup is standard or not.

**Comment by garfield69** [comment]
oh, just spotted you already supplied TPB site link.
so open a terminal and please post the output of
`curl -vvv https://thepiratebay.org/`
and 
`openssl s_client -connect thepiratebay.org:443` 
and do the same for kickasstorrents's site link

**Comment by thestaffie** [comment]
donovan@maraisdl:~$ curl -vvv https://thepiratebay.org/
*   Trying 0.0.0.0...
* TCP_NODELAY set
* connect to 0.0.0.0 port 443 failed: Connection refused
*   Trying ::...
* TCP_NODELAY set
* connect to :: port 443 failed: Connection refused
* Failed to connect to thepiratebay.org port 443: Connection refused
* Closing connection 0
curl: (7) Failed to connect to thepiratebay.org port 443: Connection refused
donovan@maraisdl:~$ openssl s_client -connect thepiratebay.org:44
140276955988416:error:0200206F:system library:connect:Connection refused:../crypto/bio/b_sock2.c:110:
140276955988416:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:111:
140276955988416:error:0200206F:system library:connect:Connection refused:../crypto/bio/b_sock2.c:110:
140276955988416:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:111:
connect:errno=111
donovan@maraisdl:~$ 


curl

> openssl s_client -connect thepiratebay.org:443

donovan@maraisdl:~$ curl -vvv https://thepiratebay.org/
*   Trying 0.0.0.0...
* TCP_NODELAY set
* connect to 0.0.0.0 port 443 failed: Connection refused
*   Trying ::...
* TCP_NODELAY set
* connect to :: port 443 failed: Connection refused
* Failed to connect to thepiratebay.org port 443: Connection refused
* Closing connection 0
curl: (7) Failed to connect to thepiratebay.org port 443: Connection refused
donovan@maraisdl:~$ openssl s_client -connect thepiratebay.org:443
140430116598208:error:0200206F:system library:connect:Connection refused:../crypto/bio/b_sock2.c:110:
140430116598208:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:111:
140430116598208:error:0200206F:system library:connect:Connection refused:../crypto/bio/b_sock2.c:110:
140430116598208:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:111:
connect:errno=111
donovan@maraisdl:~$

**Comment by garfield69** [comment]
you are being blocked by your isp or something is wrong with the DNS you are using.
the ip address for TPB should not be 0.0.0.0, which is why jackett is failing too.
if you use the browser on your ubuntu to browse https://thepiratebay.org/ what do you get?

**Comment by thestaffie** [comment]
![image](https://user-images.githubusercontent.com/52735977/61694573-0972cf80-ad32-11e9-9f45-e120f6c58b61.png)

ay way around it???

**Comment by thestaffie** [comment]
> ![image](https://user-images.githubusercontent.com/52735977/61694573-0972cf80-ad32-11e9-9f45-e120f6c58b61.png)
> 
> ay way around it???

at home I hget 
![image](https://user-images.githubusercontent.com/52735977/61694720-58b90000-ad32-11e9-8447-6cb7cacedbe3.png)

**Comment by thestaffie** [comment]
so TPB is blocked at work but fine a home
that's(home) whwre I need it to work...

**Comment by garfield69** [comment]
clearly, you should not be using jackett at work, its against company policy, your work PC may be monitored and you could get into trouble with your boss.
at home, if the browser is  working then so should jackett.

**Comment by thestaffie** [comment]
That was just a test
Most changes and config dome via remote desktop onto my home devices

**Comment by thestaffie** [comment]
> clearly, you should not be using jackett at work, its against company policy, your work PC may be monitored and you could get into trouble with your boss.
> at home, if the browser is working then so should jackett.

It does not work at home either

**Comment by thestaffie** [comment]
> oh, just spotted you already supplied TPB site link.
> so open a terminal and please post the output of
> `curl -vvv https://thepiratebay.org/`
> and
> `openssl s_client -connect thepiratebay.org:443`
> and do the same for kickasstorrents's site link

after following y9out instruction above I did run he command s on my home box an received

- donovan@maraisdl:~$ curl -vvv https://thepiratebay.org/
- *   Trying 0.0.0.0...
- * TCP_NODELAY set
- * connect to 0.0.0.0 port 443 failed: Connection refused
- *   Trying ::...
- * TCP_NODELAY set
- * connect to :: port 443 failed: Connection refused
- * Failed to connect to thepiratebay.org port 443: Connection refused
- * Closing connection 0
- curl: (7) Failed to connect to thepiratebay.org port 443: Connection refused
- donovan@maraisdl:~$ 
- donovan@maraisdl:~$ 
- donovan@maraisdl:~$ curl -vvv https://katcr.co/
- *   Trying 91.212.150.157...
- * TCP_NODELAY set
- * Connected to katcr.co (91.212.150.157) port 443 (#0)
- * ALPN, offering h2
- * ALPN, offering http/1.1
- * successfully set certificate verify locations:
- *   CAfile: /etc/ssl/certs/ca-certificates.crt
-   CApath: /etc/ssl/certs
- * (304) (OUT), TLS handshake, Client hello (1):
- * (304) (IN), TLS handshake, Server hello (2):
- * TLSv1.2 (IN), TLS handshake, Certificate (11):
- * TLSv1.2 (IN), TLS handshake, Server key exchange (12):
- * TLSv1.2 (IN), TLS handshake, Server finished (14):
- * TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
- * TLSv1.2 (OUT), TLS change cipher, Client hello (1):
- * TLSv1.2 (OUT), TLS handshake, Finished (20):
- * TLSv1.2 (IN), TLS handshake, Finished (20):
- * SSL connection using TLSv1.2 / ECDHE-RSA-AES256-GCM-SHA384
- * ALPN, server did not agree to a protocol
- * Server certificate:
- *  subject: OU=Domain Control Validated; OU=PositiveSSL Multi-Domain; CN=katcr.co
- *  start date: Nov 20 00:00:00 2017 GMT
- *  expire date: Nov 20 23:59:59 2019 GMT
- *  subjectAltName: host "katcr.co" matched cert's "katcr.co"
- *  issuer: C=GB; ST=Greater Manchester; L=Salford; O=COMODO CA Limited; CN=COMODO RSA Domain Validation Secure Server CA
- *  SSL certificate verify ok.
- > GET / HTTP/1.1
- > Host: katcr.co
- > User-Agent: curl/7.58.0
- > Accept: */*
- > 
- < HTTP/1.1 200 OK
- < Date: Tue, 23 Jul 2019 08:34:35 GMT
- < Server: Apache/2.4.10
- < Vary: Accept-Encoding
- < Cache-Control: max-age=300
- < Expires: Tue, 23 Jul 2019 08:35:08 GMT
- < X-Frame-Options: sameorigin
- < X-XSS-Protection: 1; mode=block
- < X-Content-Type-Options: nosniff
- < Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
- < X-Frame-Options: SAMEORIGIN
- < Age: 267
- < X-Cache: HIT from katcr.co
- < Content-Length: 5581
- < Connection: close
- < Content-Type: text/html; charset=UTF-8
- < 
- <!DOCTYPE html>
- <html lang="en-us">
- <head>
-     <meta charset="UTF-8">
- 	<noscript>
- 	<meta http-equiv="refresh" content="0; url=https://katcr.co/ikickass/" />
- 	</noscript>
-     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-     <meta http-equiv="Content-Style-Type" content="text/css">
-     <meta name="viewport" content="width=device-width, initial-scale=1">
-     <meta http-equiv="cache-control" content="no-cache, must-revalidate">
-     <meta http-equiv="expires" content="0">
-     <meta http-equiv="pragma" content="no-cache">
-     <meta http-equiv="X-UA-Compatible" content="IE=Edge">
-     <title>KAT - Kickass Torrents</title>
-     <link rel="shortcut icon" type="image/x-icon" href="https://interscloud-631f.kxcdn.com/kloud/favicon.ico" />
-     <meta name="description" content="Search and discuss new and favorite TV shows &amp; TV series, movies, music and games.">
-     <link rel="search" type="application/opensearchdescription+xml" title="KickassTorrents Torrent Search" href="#">
-     <meta property="fb:app_id" content="">
-     <meta name="apple-mobile-web-app-capable" content="yes">
-     <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
-     <link rel="stylesheet" type="text/css" href="/community/torrents/show/css/welcomelanding.css" charset="utf-8">
-     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-   </head>
-   <body class="landing-page">
-     <div class="header" style="overflow: hidden;">
-       <div class="switch">
-         <a href="/new/full/">
-           Switch to full page
-         </a>
-       </div>
-       <div class="login">
-       <div class="login">
-         <a href="/show/community/index.php?action=login">
-           <i class="ka ka-user"></i>
-           <span>Sign In</span>
-         </a>
-       </div>
-       </div>
-     </div>
-     <div class="content">
-       <a href="/new/full/">
-         <div class="logo-sign">
-         </div>
-       </a>
-       <a href="/new/full/">
-         <div class="logo-text">
-         </div>
-       </a>
-       <div class="search-input">
- 
-         <form action="/usearch/torrents-search.php" method="get" accept-charset="utf-8" onsubmit="return doSearch(this.search.value);">
-           <input id="contentSearch" type="text" name="q" value="" autocomplete="off" placeholder="Search query" required>
-           <div id="searchTool">
-             <a title="Advanced search disabled" href=""><i class="ka ka-settings"></i></a>
-             <button title="Search" type="submit" value="Torrent Search" onfocus="this.blur();" onclick="this.blur();"><i class="ka ka-search"></i></button>
-           </div>
-         </form>
-         
-       </div>
-       <ul class="nav">
-         <li><a href="/category/movies/page/">Movies</a></li> 
-         <li><a href="/category/tv/page/">TV</a></li>
-         <li><a href="/category/games/page/">Games</a></li>
-         <li><a href="/category/music/page/">Music</a></li>
-         <li><a href="/category/latest/page/">New</a></li>
- 		<li><a href="/new/">Browse</a></li>
-       </ul>
-     </div>
-     <div class="bottom-menu-block">
-       <ul class="social-buttons">
-         <li>
-           <a href="https://twitter.com/KATCommunity" class="twitter-follow-button" data-show-count="true">Follow @KATCommunity</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
-         </li>
-         <li>
-           <div id="fb-root"></div>
-           <script>(function(d, s, id) {
-             var js, fjs = d.getElementsByTagName(s)[0];
-             if (d.getElementById(id)) return;
-             js = d.createElement(s); js.id = id;
-             js.src = "https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8";
-             fjs.parentNode.insertBefore(js, fjs);
-           }(document, 'script', 'facebook-jssdk'));</script>
-           <div class="fb-like" data-href="https://www.facebook.com/KAT-Community-153971625028766" data-layout="button_count" data-action="like" data-show-faces="false" data-share="false" data-size="small"></div>
-         </li>
-       </ul>
-       <ul class="bottom-menu">
-         <li>
-           <a href="





show/community/">
-             <i class="ka ka-community"></i>Community
-           </a>
-         </li>
-         <li>
-           <a href="https://katcr.co/show/dmca/">
-             <i class="ka ka-settings"></i>DMCA
-           </a>
-         </li>
-         <li>
-           <a href="https://katcr.co/show/tos/">
-             <i class="ka ka-settings"></i>TOS
-           </a>
-         </li>
-       </ul>
-     </div>
-     <script type="text/javascript" charset="utf-8">
-       $(document).ready(function() {  $("#contentSearch").focus(); });
-     </script>





donovan@maraisdl:~$ openssl s_client -connect thepiratebay.org:443
140509943394752:error:0200206F:system library:connect:Connection refused:../crypto/bio/b_sock2.c:110:
140509943394752:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:111:
140509943394752:error:0200206F:system library:connect:Connection refused:../crypto/bio/b_sock2.c:110:
140509943394752:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:111:
connect:errno=111
donovan@maraisdl:~$ 
donovan@maraisdl:~$ 
donovan@maraisdl:~$ 
donovan@maraisdl:~$ openssl s_client -connect katcr.co/g:443
140082575417792:error:2008F002:BIO routines:BIO_lookup_ex:system lib:../crypto/bio/b_addr.c:704:Name or service not known
connect:errno=2
donovan@maraisdl:~$ 

-     <script type="application/ld+json">
-       {
-         "@context": "https://schema.org",
-         "@type": "Organization",
-         "name": "KickassTorrents",
-         "url": "https://katcr.co/",
-         "logo": "https://katcr.co/community/show/images/kickasslogo.png",
-         "sameAs": [
-           "https://www.facebook.com/KAT-Community-153971625028766",
-           "https://twitter.com/KATCommunity"
-         ]
-       }
-     </script>
-     <script type="application/ld+json">
-       {
-         "@context": "https://schema.org",
-         "@type": "WebSite",
-         "url": "https://katcr.co",
-         "potentialAction": {
-           "@type": "SearchAction",
-           "target": "https://katcr.co/usearch/torrents-search.php?q={q}",
-           "query-input": {
-             "@type": "PropertyValueSpecification",
-             "valueRequired": true,
-             "valueName": "q"
-           }
-         }
-       }
-     </script>
-   </body>
- * Closing connection 0
- * TLSv1.2 (OUT), TLS alert, Client hello (1):
- </html>donovan@maraisdl:~$ 
-

**Comment by garfield69** [comment]
is your ubuntu OS that you have at home built from a work image, and is thus running the fortiguard web filter?

**Comment by thestaffie** [comment]
There's no fortigaurd on home ubuntu...  got pB to work... now just KAT

**Comment by garfield69** [comment]
how did you get TPB to work?

**Comment by thestaffie** [comment]
changed the url

**Comment by thestaffie** [comment]
but KAT still fails

**Comment by garfield69** [comment]
with what error message from jackett?

**Comment by thestaffie** [comment]
![image](https://user-images.githubusercontent.com/52735977/61699914-14caf880-ad3c-11e9-980f-6093448f1a1b.png)

**Comment by thestaffie** [comment]
When | Level | Message
-- | -- | --
Jul 23, 2019 11:21:37 AM | Error | Jackett.Common.IndexerException: Exception (kickasstorrent): Request headers must contain only ASCII characters. ---> System.Net.Http.HttpRequestException: Request headers must contain only ASCII characters.    at System.Net.Http.HttpConnection.WriteStringAsync(String s)    at System.Net.Http.HttpConnection.WriteHeadersAsync(HttpHeaders headers, String cookiesFromContainer)    at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)    at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)    at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)    at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)    at CloudFlareUtilities.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)    at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)    at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs:line 241    at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/WebClient.cs:line 114    at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 445    at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1230    at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 331    --- End of inner exception stack trace ---    at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 351    at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 815    at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/appveyor/projects/jackett/src/Jackett.Common/Services/IndexerManagerService.cs:line 222    at Jackett.Server.Controllers.IndexerApiController.Test() in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 128    at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()    at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)    at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)    at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 29

**Comment by garfield69** [comment]
Ahh.
that is a duplicate of #5594 
nothing you can do about this until the C# devs get around to fixing it.
ITMT  I suggest using the katcrs indexer or the kickasstorrent(kat.li) indexer instead.
